hls4ml/model/layers.py

```diff
@@ -560,6 +560,7 @@ def initialize(self):
         if self.model.config.is_resource_strategy(self) and self.model.config.backend.name in [
             'Vivado',
             'VivadoAccelerator',
+            'Catapult',
```

Comment:
Truthfully, I think this needs to go away, also for Vivado. The standard representation should not have model-specific hooks. These changes should be made in the backend.

Reply:
I made a general issue about this:  https://github.com/fastmachinelearning/hls4ml/issues/957